sdk/resourcemanager/Azure.ResourceManager/src/Generated/Extensions/ResourceGroupExtensions.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager
+{
+    /// <summary> A class to add extension methods to ResourceGroup. </summary>
+    public static partial class ResourceGroupExtensions
```

Comment:
nit: We should update autorest to skip this file if empty. Probably post preview. But I guess most RP will have RG and Sub extensions then.